helidon-cli/harness/pom.xml

```diff
@@ -43,6 +43,18 @@
             <artifactId>spotbugs-annotations</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>io.helidon.build-tools</groupId>
+            <artifactId>helidon-utils</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.helidon.build-tools</groupId>
+            <artifactId>helidon-utils</artifactId>
+            <version>${project.version}</version>
+            <type>test-jar</type>
```

Comment:
I'm not a fan of test-jar as that breaks the "-DskipTests" builds since the test-jar dependency resolution fails.
Can we make it distinct module ?

Reply:
I do `-DskipTests` here all the time; how does this break for you?